AzureCalling/app/src/main/res/layout/activity_call.xml

```diff
@@ -113,48 +118,90 @@
         </ImageButton>
     </LinearLayout>
 
+    <LinearLayout
+        android:id="@+id/in_lobby_waiting_overlay"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="#CC000000"
+        android:gravity="center"
+        android:orientation="vertical"
+        app:layout_constraintBottom_toBottomOf="@+id/previewHolder"
+        app:layout_constraintEnd_toEndOf="@+id/previewHolder"
+        app:layout_constraintStart_toStartOf="@+id/previewHolder"
+        app:layout_constraintTop_toTopOf="@+id/previewHolder"
+        tools:visibility="invisible">
+
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginLeft="16dp"
+            android:layout_marginTop="212dp"
+            android:layout_marginRight="16dp"
+            android:layout_marginBottom="10dp"
+            android:gravity="center"
+            android:letterSpacing="0.01"
+            android:text="@string/in_lobby_waiting_message_title"
+            android:textColor="@color/defaultWhite"
+            android:textSize="24sp"
+            android:textStyle="bold" />
+
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginLeft="16dp"
+            android:layout_marginRight="16dp"
+            android:layout_marginBottom="24dp"
+            android:gravity="center"
+            android:letterSpacing="0.01"
+            android:text="@string/in_lobby_waiting_message_description"
+            android:textColor="@color/defaultWhite"
+            android:textSize="20sp" />
+
+    </LinearLayout>
+
     <LinearLayout
         android:id="@+id/call_hangup_overlay"
         android:layout_width="0dp"
         android:layout_height="0dp"
-        android:background="#66000000"
+        android:background="#CC000000"
```

Comment:
Is this a change that would be applicable in the main branch as well?

Reply:
yes, sent new PR to main.